include/rnp/rnp.h

```diff
@@ -551,6 +551,17 @@ RNP_API rnp_result_t rnp_request_password(rnp_ffi_t        ffi,
                                           const char *     context,
                                           char **          password);
 
+/**
+ * @brief Set timestamp, used in all operations instead of system's time.
```

Comment:
It would be nice to have more detailed description where this timestamp can be used, like new signatures, generated key timestamps, etc.

Reply:
Thanks, updated.